config.mak.uname

```diff
@@ -17,6 +17,13 @@ ifdef MSVC
 	uname_O := Windows
 endif
 
+
+ifdef MSVC
+GIT-MSVC-GEN: ./compat/vcbuild/find_vs_env.bat
+	@./compat/vcbuild/find_vs_env.bat | sed 's|\\|/|g' >GIT-MSVC-GEN
+-include GIT-MSVC-GEN
+endif
+
```

Comment:
@dscho From what I can tell, a generated included file causes an implicit dependency.  Make automatically takes care of forcing the file to be generated.